src/js/button.js

```diff
@@ -45,6 +45,7 @@ class Button extends ClickableComponent {
     attributes = assign({
       type: 'button', // Necessary since the default button type is "submit"
       'aria-live': 'polite' // let the screen reader user know that the text of the button may change
+      'tabindex' : 0 // add tabindex for keyboard accessibility to buttons
```

Comment:
This specifically doesn't add `tabindex=0` to native `button` elements, where it isn't required. The `ClickableComponent` does add `tabindex=0` where non-button elements (typically `div`s) are used as buttons, otherwise they won't receive keyboard focus. I believe I've moved all the `extends Button`s over to `extends ClickableComponent`s where appropriate, so this all works out correctly.

If I've missed a control, or controls that aren't part of the video.js core extend a button and then override its element, then the test which produce the warning should also add a `tabindex=0`, for backward compatibility. Is this what you're referring to, @marguinbc, or are you expecting to see `tabindex` on native `button` elements still?